test/test-server/connect_handler.go

```diff
@@ -298,7 +298,7 @@ func handleCustomPluginUploadUrl(t *testing.T) http.HandlerFunc {
 			uploadUrl := connectcustompluginv1.ConnectV1PresignedUrl{
 				ContentFormat: PtrString("ZIP"),
 				UploadId:      PtrString("e53bb2e8-8de3-49fa-9fb1-4e3fd9a16b66"),
-				UploadUrl:     PtrString("https://api.confluent.cloud/connect/v1/dummy-presigned-url"),
+				UploadUrl:     PtrString("http://127.0.0.1:2048/connect/v1/dummy-presigned-url"),
```

Comment:
Can you use the constant here instead in case we decide to change the port number in the future?